websites/W/Wolfy/presence.ts

```diff
@@ -27,6 +26,85 @@ function getTimestamps(audioTime: number, audioDuration: string): number[] {
 	];
 }
 
+async function addJoinGameButton(presenceData: PresenceData, gameId: string) {
+	if (!(await presence.getSetting("joinGameButton"))) return;
+	if (!presenceData.buttons) {
+		presenceData.buttons = [
+			{
+				label: `(${
+					document.querySelector("div.Header_timer__36MsP")?.textContent
+				}) Rejoindre la partie`,
+				url: `https://wolfy.fr/game/${gameId}`
+			}
+		];
+	} else {
+		presenceData.buttons[1] = {
+			label: `(${
+				document.querySelector("div.Header_timer__36MsP")?.textContent
+			}) Rejoindre la partie`,
+			url: `https://wolfy.fr/game/${gameId}`
+		};
+	}
+}
+
+async function addVisitProfilButton(
+	presenceData: PresenceData,
+	username: string
+) {
+	if (!(await presence.getSetting("visitProfileButton"))) return;
+	if (!presenceData.buttons) {
+		presenceData.buttons = [
+			{
+				label: `Visiter le profil de ${username}`,
+				url: `https://wolfy.fr/leaderboard/${username}`
+			}
+		];
+	} else {
+		presenceData.buttons[1] = {
+			label: `Visiter le profil de ${username}`,
+			url: `https://wolfy.fr/leaderboard/${username}`
+		};
+	}
```

Comment:
You can only have 32 characters on the labels
So this may not be a good idea

Reply:
Thx, i just commited a fix about that.

If the label is longer than 32 characters i cut it now.